docs/docs/.vuepress/configs/sidebarConfig.js

```diff
@@ -112,7 +104,16 @@ const referenceNav = [
     ],
   },
   "/limits/",
-  "/privacy-and-security/"
+  {
```

Comment:
This was a small bug I found — security section wasn't rendering anything beyond `/privacy-and-security/` the way it was previously configured.

Reply:
I was actually fixing this in another branch too, until I got hit with that refresh bug. I'll just clean up the merge conflict if there is one.